content/en/dora_metrics/_index.md

```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
 
 You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [DORA Metrics page][8].
 
-Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards and alerted on using [metric monitors][9].
+Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][9].
+
+The metrics can also be queried with the [Query timeseries points][10] and [Query timeseries data across multiple products][11] API endpoints.
+
+The metrics provided by DORA Metrics are:
+- `dora.deployments.count` of type `count`. Used for Deployment Frequency.
+- `dora.change_lead_time` of type `distribution`. Contains the age in `seconds` of the git commits at the time of deployment.
+- `dora.incidents_impact` of type `count`. Tracks the services or teams impacted by incidents. Used for Change Failure Rate with the formula `dora.incidents_impact / dora.deployments.count`. A big time rollup of at least 1 week is recommended to account for time difference between deployments and when the impact starts.
+- `dora.time_to_restore` of type `distribution`. Contains the time in `seconds` between the incident's `started_at` and `finished_at`.
+
+All the metrics contain the following tags when available:
+- `service`
+- `team`
+- `env`
+- `repository_id`
+
+**Note**: The `severity` tag is available for the `dora.incidents_impact` and `dora.time_to_restore` metrics, if provided through the API.
+
+### Deployment and Incident events
+
+DORA Metrics also provides individual `deployment`, `incident`, and `incident_finished` events in [Event Management][12] with `source:software_delivery_insights`.
+
+The events can be queried and visualized with [Event Management][13].
```

Comment:
"The events can be queried and visualized with the [Events Explorer](https://docs.datadoghq.com/service_management/events/explorer/)."